src/app/common/transactions/bitcoin/use-generate-bitcoin-tx.ts

```diff
@@ -20,7 +20,14 @@ interface GenerateNativeSegwitTxValues {
   recipients: TransferRecipient[];
 }
 
-export function useGenerateUnsignedNativeSegwitTx() {
+interface UseGenerateUnsignedNativeSegwitTxProps {
+  throwError?: boolean;
+}
+
+// temp arg before refactoring all flows to new design
```

Comment:
Is there an issue for this work? There's none in the commit messages either. 

It would be nice to mention an issue here / in the commit message so we don't forget about this